ui/app/serializers/pki/issuer.js

```diff
@@ -37,10 +38,24 @@ export default class PkiIssuerSerializer extends ApplicationSerializer {
   normalizeItems(payload) {
     if (payload.data) {
       if (payload.data?.keys && Array.isArray(payload.data.keys)) {
-        return payload.data.keys.map((issuer_id) => ({
-          issuer_id,
-          ...payload.data.key_info[issuer_id],
-        }));
+        return payload.data.keys.map((key) => {
+          if (key.shouldShowIssuerMetaData) {
+            const { certificate, issuer_id, isRoot } = key;
+            const parsedCert = parseCertificate(certificate);
+
+            return {
+              issuer_id,
+              ...payload.data.key_info[issuer_id],
+              parsed_certificate: parsedCert,
```

Comment:
Where is the data from `parsed_certificate` used?

Reply:
For common name! :)